tests/features/bootstrap/FeatureContext.php

```diff
@@ -386,6 +386,42 @@ public function assertFieldOptionSelected($option, $select) {
   }
 
   /**
+   * Checks that a certain radio input is checked in a specific field.
+   *
+   * @param string $radio
+   *   The label of the radio input to find.
+   * @param string $field
+   *   The label of the field the radio is part of.
+   *
+   * @throws \Exception
+   *   Thrown when the field or the radio is not found, or if the radio is not
+   *   checked.
+   *
+   * @Then the radio :radio from field :field should be checked
+   */
+  public function assertFieldRadioSelected($radio, $field) {
```

Comment:
Ok for me, but this was actually changed by you in the previous QA.
Commit `79b7b542cb729041a4bdcaafa22b9d360a51e142`:
"ISAICP-2991: In usual terminology a radio button is 'selected' rather than 'checked'."
What to do?